Changelogs for Digital Twin Manager Microservice 1025.0.0 (#4494)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1025-0-0-improved-performance-when-creating-updating-linked-series-opposites.md

---

> date: ""

Reviewer: ```suggestion
date: "2026-04-09"
```

> title: "Improved performance when creating/updating linked series opposites."

Reviewer 2: ```suggestion
title: "Improved performance when creating or updating linked data points"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2676"
version: "1025.0.0"
---
Improved performance when creating and updating linked series opposites
by consolidating device and opposite lookups into a single inventory

> call.

Reviewer 2: ```suggestion
When creating or updating linked data points using API, the endpoint previously made separate inventory API calls to look up the source device and resolve the corresponding opposite series entries. This increased latency, particularly in environments with large asset hierarchies containing many linked data points.

Digital Twin Manager now consolidates the device and opposite series lookups into a single inventory call. The reduced number of backend requests results in faster response times when you create or update linked data points. No action is required on existing installations.
```

> When creating or updating linked data points using API, the endpoint previously made separate inventory API calls to look up the source device and resolve the corresponding opposite series entries. This increased latency, particularly in environments with large asset hierarchies containing many linked data points.

Reviewer 3: ```suggestion
When creating or updating linked data points via the API, the endpoint previously made separate inventory API calls to look up the source device and resolve the corresponding opposite-series entries. This increased latency, particularly in environments with large asset hierarchies containing many linked data points.
```

> When creating or updating linked data points via the API, the endpoint previously made separate inventory API calls to look up the source device and resolve the corresponding opposite-series entries. This increased latency, particularly in environments with large asset hierarchies containing many linked data points.

Reviewer 3: ```suggestion
When creating or updating linked data points via the API, the endpoint previously made separate Inventory API calls to look up the source device and resolve the corresponding opposite-series entries. This increased latency, particularly in environments with large asset hierarchies containing many linked data points.
```

> Digital Twin Manager now consolidates the device and opposite series lookups into a single inventory call. The reduced number of backend requests results in faster response times when you create or update linked data points. No action is required on existing installations.

Reviewer 3: ```suggestion
The Digital Twin Manager now consolidates the device and opposite series lookups into a single inventory call. The reduced number of backend requests results in faster response times when you create or update linked data points. No action is required on existing installations.
```

> The Digital Twin Manager now consolidates the device and opposite series lookups into a single inventory call. The reduced number of backend requests results in faster response times when you create or update linked data points. No action is required on existing installations.

Reviewer 3: ```suggestion
The Digital Twin Manager now consolidates the device and opposite series lookups into a single Inventory API call. Fewer backend requests result in faster response times when you create or update linked data points. No action is required on existing installations.
```
